### ISSUE TITLE
Should adding a `NOT NULL` column with a default warn in PG >=11?

### ISSUE THREAD

**Issue by sbdchd** [sbdchd/squawk#136]
```sql
ALTER TABLE "foo_tbl" ADD COLUMN "bar_col" TEXT DEFAULT 'buzz' NOT NULL;
```

```
stdin:2:1: warning: adding-not-nullable-field
   1 | ALTER TABLE "foo_tbl" ADD COLUMN "bar_col" TEXT DEFAULT 'buzz' NOT NULL;
  note: Adding a NOT NULL field requires exclusive locks and table rewrites.
  help: Make the field nullable.
```

related: https://github.com/sbdchd/squawk/issues/109

**Comment by sbdchd** [comment]
From the docs:

> From PostgreSQL 11, adding a column with a constant default value no longer means that each row of the table needs to be updated when the ALTER TABLE statement is executed. Instead, the default value will be returned the next time the row is accessed, and applied when the table is rewritten, making the ALTER TABLE very fast even on large tables.

https://www.postgresql.org/docs/13/ddl-alter.html

**Comment by sbdchd** [comment]
fixed in latest
